ui-tests/pom.xml

```diff
@@ -33,14 +33,29 @@
         <artifactId>jenkins-core</artifactId>
         <version>${jenkins.version}</version>
       </dependency>
+	<!-- need to match any BOM imports from ATH due to https://issues.apache.org/jira/browse/MNG-5761 -->
+      <dependency>
+        <groupId>io.netty</groupId>
+        <artifactId>netty-bom</artifactId>
+        <version>4.1.91.Final</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+      <dependency>
```

Comment:
there was a mix of different jackson versions (core databind and some others) that caused CNFE.